articles/language/type-model.md

```diff
@@ -280,6 +261,9 @@ defining the implementation of the new operation.
 Thus, functors can perform more complex functions than
 traditional higher-level functions.
 
+Functors do not have a representation in the Q# type system. 
```

Comment:
It might be more clear to note that functors are not Q# values, and thus do not have types?

Reply:
I'd like to postphone having to have a discussion on what functors are/could/ or will be.